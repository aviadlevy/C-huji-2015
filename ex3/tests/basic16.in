16
0 0 0 0 0 0 0 0 0 0 0 0 6 9 0 4
0 0 0 0 0 0 0 15 8 0 0 0 0 0 0 0
0 0 0 0 0 0 0 13 0 0 0 0 0 1 0 0
0 0 0 0 0 3 0 12 0 0 1 15 5 0 0 14
7 0 15 4 9 5 0 14 0 0 3 0 10 0 6 0
0 9 0 0 3 7 0 4 0 0 0 6 0 13 0 2
8 14 0 1 6 10 0 11 15 7 0 0 12 16 0 0
0 13 0 0 0 0 15 1 0 11 2 0 0 7 4 3
14 16 2 0 5 0 11 0 0 0 0 0 9 6 12 0
9 0 0 0 1 12 0 2 0 10 5 8 3 4 13 16
5 0 0 0 15 0 16 6 0 9 11 0 0 0 10 1
0 6 0 0 0 9 3 0 2 12 0 0 15 14 0 11
0 3 0 0 0 0 0 0 13 1 0 0 4 12 11 0
6 11 8 0 0 15 1 7 4 5 0 3 0 2 0 10
1 7 0 13 12 0 4 0 11 14 16 0 0 0 3 0
4 0 0 0 11 0 0 3 0 0 0 0 0 5 0 0