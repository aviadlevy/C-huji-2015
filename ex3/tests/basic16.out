16
3 1 12 15 14 11 7 16 5 2 10 13 6 9 8 4
2 5 6 14 10 1 9 15 8 4 7 11 13 3 16 12
10 8 16 7 4 6 5 13 14 3 12 9 11 1 2 15
11 4 13 9 2 3 8 12 16 6 1 15 5 10 7 14
7 2 15 4 9 5 13 14 1 16 3 12 10 11 6 8
16 9 11 5 3 7 12 4 10 8 14 6 1 13 15 2
8 14 3 1 6 10 2 11 15 7 13 4 12 16 9 5
12 13 10 6 8 16 15 1 9 11 2 5 14 7 4 3
14 16 2 8 5 4 11 10 3 13 15 1 9 6 12 7
9 15 7 11 1 12 14 2 6 10 5 8 3 4 13 16
5 12 4 3 15 13 16 6 7 9 11 14 2 8 10 1
13 6 1 10 7 9 3 8 2 12 4 16 15 14 5 11
15 3 14 2 16 8 10 5 13 1 6 7 4 12 11 9
6 11 8 12 13 15 1 7 4 5 9 3 16 2 14 10
1 7 5 13 12 2 4 9 11 14 16 10 8 15 3 6
4 10 9 16 11 14 6 3 12 15 8 2 7 5 1 13
